ui/components/app/collectibles-items/collectibles-items.js

```diff
@@ -0,0 +1,222 @@
+                    <Typography
+                      color={COLORS.BLACK}
+                      variant={TYPOGRAPHY.H4}
+                      margin={[0, 0, 0, 2]}
+                    >
+                      {`${collectionName} (${collectibles.length})`}
+                    </Typography>
+                  </Box>
+                  <Box alignItems={ALIGN_ITEMS.FLEX_END}>
+                    <i
+                      className={`fa fa-lg fa-chevron-${
+                        isExpanded ? 'down' : 'right'
+                      }`}
+                    />
+                  </Box>
+                </Box>
+                {isExpanded ? (
+                  <Box display={DISPLAY.FLEX} flexWrap={FLEX_WRAP.WRAP}>
+                    {collectibles.map((collectible, i) => {
+                      const {
+                        image,
+                        address,
+                        tokenId,
+                        backgroundColor,
+                      } = collectible;
+                      const collectibleImage = getAssetImageURL(
+                        image,
+                        ipfsGateway,
+                      );
+                      return (
+                        <Box width={width} margin={1} key={`collectible-${i}`}>
+                          <div
+                            className="collectibles-items__image__wrapper"
+                            style={{
+                              backgroundColor,
+                            }}
+                          >
+                            <img
+                              onClick={() =>
+                                history.push(
+                                  `${ASSET_ROUTE}/${address}/${tokenId}`,
+                                )
+                              }
+                              className="collectibles-items__image"
+                              src={collectibleImage}
+                            />
+                          </div>
+                        </Box>
+                      );
+                    })}
+                  </Box>
+                ) : null}
+              </div>
+            );
+          })}
+          <Box
+            marginTop={6}
+            flexDirection={FLEX_DIRECTION.COLUMN}
+            justifyContent={JUSTIFY_CONTENT.CENTER}
+          >
+            <Typography
+              color={COLORS.UI3}
+              variant={TYPOGRAPHY.H5}
+              align={TEXT_ALIGN.CENTER}
+            >
+              {t('missingNFT')}
+            </Typography>
+            <Box
+              alignItems={ALIGN_ITEMS.CENTER}
+              justifyContent={JUSTIFY_CONTENT.CENTER}
+            >
+              <Box
+                className="collectibles-items__link"
+                justifyContent={JUSTIFY_CONTENT.FLEX_END}
+              >
+                {useCollectibleDetection ? (
+                  <Button type="link" onClick={onRefreshList}>
+                    {t('refreshList')}
```

Comment:
Instead of "Import NFTs", should this text be consistent with what is shown if there are no NFTs ("Add NFT")?